src/screens/Home/Home.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { Anchor, Box, Grid, Image, Text } from 'grommet'
+import Kelp from 'images/kelp.png'
+import Title from 'images/title.svg'
+import Zooniverse from 'images/zooniverse.png'
+import ResponsiveImage from 'components/ResponsiveImage'
+import styled from 'styled-components'
+import ChooseLocation from './components/ChooseLocation'
+
+const StyledText = styled(Text)`
+  font-family: Neuton;
+`
+
+export default function Home() {
+  return (
+    <Box background='sand' direction='row' fill>
+      <Grid
+        rows={['1/2', '1/2']}
+        columns={['1/4', '3/4']}
+        areas={[
+          { name: 'banner', start: [0, 0], end: [0, 1] },
+          { name: 'intro', start: [1, 0], end: [1, 1] },
+          { name: 'body', start: [1, 1], end: [1, 1] }
+        ]}
+      >
+        <Box gridArea='banner' border={{ color: 'kelp', side: 'right', size: 'large' }}>
+          <Image gridArea='banner' fit='cover' src={Kelp} />
+        </Box>
+
+        <Box
+          gridArea='intro'
+          margin={{ horizontal: 'medium', top: 'auto' }}
+          width={{ max: '40rem' }}
+        >
+          <ResponsiveImage height='1.25rem' src={Zooniverse} />
```

Comment:
We have the logotype available in the `@zooniverse/react-components` library as a component that has the needed accessibility applied.